test-framework/devmode-test-utils/src/main/java/io/quarkus/test/devmode/util/DevModeTestUtils.java

```diff
@@ -22,6 +22,16 @@
 
 public class DevModeTestUtils {
```

Comment:
Should we consider renaming it to something like DevModeClient? Doesn't have to be a part of this PR.

Reply:
Good idea. The change touches so many files I'll do it as part of this PR to keep the churn to a minimum.